internal/config/alias_config.go

```diff
@@ -42,3 +42,22 @@ func (a *AliasConfig) Delete(alias string) error {
 	// TODO when we get to gh alias delete
 	return nil
 }
+
+func (a *AliasConfig) All() map[string]string {
+	out := map[string]string{}
+
+	if a.Empty() {
+		return out
+	}
+
+	for i := 0; i < len(a.Root.Content); i += 2 {
+		if i+1 == len(a.Root.Content) {
```

Comment:
Could the condition inside the `for` loop be adjusted so that we don't need this extra check?
```suggestion
	for i := 0; i < len(a.Root.Content)-1; i += 2 {
```